revolv/templates/base/partials/foot.html

```diff
@@ -51,15 +51,15 @@
           people and communities to invest collectively in renewable energy.
           All donations are tax-deductible.
         </p>
-		<dl>
-			<dd><a href="https://old.re-volv.org/">Visit our old website</a></dd>
-		</dl>
-		<dl>
-			<dt><a href="http://goo.gl/forms/S1TEKTNgNz">How do you like our new website?</a></dt>
-		</dl>
-		<p>
-			Take a short survey and let us know. We would love to hear your feedback!
-		</p>
+		    <dl>
```

Comment:
Hi Noah

i can understand  , These contents are added because there is no confirmation for this part implementation , This is design part and Andreas has asked to add this , Please confirm with Andreas what need to be done in place of  that  text ? 

There is no discussion about the funcitonality for that part with Andrea. Can you please discuss with Andreas for that part ?

Thanks